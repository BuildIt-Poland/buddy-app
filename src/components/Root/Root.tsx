import React from 'react';
import Container from '@material-ui/core/Container';
import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles(theme => ({
  container: {
    minWidth: theme.breakpoints.values['xs'],
  },
}));

const Root: React.FunctionComponent = () => {
  const classes = useStyles();

  return (
    <Container className={classes.container} component='main' maxWidth='md'>
      Buddy App
    </Container>
  );
};

export default Root;
