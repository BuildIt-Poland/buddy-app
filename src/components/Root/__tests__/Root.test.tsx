import React from 'react';
import { create } from 'react-test-renderer';

import Root from '../Root';

jest.mock('@material-ui/core/Container', () => 'Container');

describe('Component - Root', () => {
  test('renders correctly', () => {
    const component = create(<Root></Root>);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
