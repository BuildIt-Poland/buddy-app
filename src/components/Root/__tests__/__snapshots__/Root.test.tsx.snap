// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component - Root renders correctly 1`] = `
<Container
  className="makeStyles-container-1"
  component="main"
  maxWidth="md"
>
  Buddy App
</Container>
`;
